lib/db/structs.proto

```diff
@@ -28,6 +28,7 @@ message FileInfoTruncated {
     uint32                permissions    = 4;
     int64                 modified_s     = 5;
     int32                 modified_ns    = 11;
+    protocol.DeviceID     lastmodifiedby = 12;
```

Comment:
Or maybe we could cut it down to modified_by given we have modified_s and modified_ns